src/renderer/services/auto-updater.service.ts

```diff
@@ -44,6 +56,34 @@ export class AutoUpdaterService{
         this.ipcService.sendLazy("install-update");
     }
 
+    public getHaveBeenUpdated() : Observable<boolean>{
+        return this.ipcService.sendV2<boolean>("have-been-updated");
+    }
+
+  public async getChangelogs(): Promise<Changelog | null> {
```

Comment:
if you remove "| null" you have an error ?

Reply:
no